src/pkgmt/config.py

```diff
@@ -31,13 +30,5 @@ def load():
     if Path("pyproject.toml").exists():
         with open("pyproject.toml") as f:
             return Config(toml.load(f)["tool"]["pkgmt"], "pyproject.toml")
-    elif Path("config.yaml").exists():
-        with open("config.yaml", "r") as stream:
-            try:
-                return Config(yaml.safe_load(stream)["pkgmt"], "config.yaml")
-            except yaml.YAMLError as exc:
-                print("Error loading config.yaml file : {}".format(exc))
     else:
-        raise FileNotFoundError(
-            "Could not load config: expected pyproject.toml or config.yaml"
-        )
+        raise FileNotFoundError("Could not load config: expected pyproject.toml")
```

Comment:
Do we have test case for this part?

Reply:
yes. `test_config::test_missing_file`.
added the error string also